### ISSUE TITLE
auth: consider moving out of chaire-lib the authentication methods that are for surveys only

### ISSUE THREAD

**Issue by tahini** [chairemobilite/transition#905]
Anonymous and direct login authentication methods are more for surveys and maybe they should be moved to evolution only, to avoid accidentally enabling them for Transition.

This will require parallel developments in evolution and transition to avoid losing those functionalities and making sure the evolution can easily extend chaire-lib with its own methods.

**Comment by greenscientist** [comment]
I don't think we should do that. Keep all auth method in one location makes sense. 

Also, we eventually would probablement want some form of anonymous login if we develop projects like public dashboard and comparaison tools.
